### DISCUSSION TITLE
Printers Printing Garbage During Scanning

### DISCUSSION THREAD

**Discussion by weldingwit** [projectdiscovery/nuclei#5761] [Q&A]
Just wondering if anyone has had the same issue that I get every time I do an Internal pentest.
I have done a lot of Internal pentests and use nuclei - however I am constantly getting emails from the client reporting that their printers have gone crazy and keep printing out garbage. So just wondering if this is a known problem and if there is a specific template that is causing this. I do not want to exclude printers or their ports during scanning as they can sometimes be a gold mine.

Thanks

**Comment by princechaddha** [comment]
@weldingwit This is most likely related to the CUPS - RCE vulnerability template: [CVE-2024-47176](https://github.com/projectdiscovery/nuclei-templates/blob/main/javascript/cves/2024/CVE-2024-47176.yaml). We haven’t encountered this issue before. Could you please try running this template to confirm if that’s the case?

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
